plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```diff
@@ -481,11 +481,14 @@ export class SessionRecordingIngester {
         // eachBatchWithContext, then commits offsets for the batch.
         // the batch consumer reads from the session replay kafka cluster
         const replayClusterConnectionConfig = createRdConnectionConfigFromEnvVars(this.config)
+
+        const autoCommit = this.config.SESSION_RECORDING_USE_OFFSET_STORE ? true : false
+
         this.batchConsumer = await startBatchConsumer({
             connectionConfig: replayClusterConnectionConfig,
             groupId: this.consumerGroupId,
             topic: this.topic,
-            autoCommit: false,
+            autoCommit,
```

Comment:
@bretthoerner this is me copying your code elsewhere - i seems we have the `enable.auto.offset.store` as false in the `startBatchConsumer` code so I am assuming turning on autoCommits and calling the offsetStore thing is the right approach?